cli_tools/gce_onestep_image_import/README.md

```diff
@@ -0,0 +1,104 @@
+## Compute Engine One-step Image Import
```

Comment:
Let's ask David and connect to tech writer about this readme and corresponding public doc change.

Reply:
sounds good!